content/source/docs/enterprise-beta/guides/practices_part1.html.md

```diff
@@ -63,11 +63,11 @@ Workspace Contributors want a simple workflow to submit changes to a workspace a
 
 Workspace contributors are often already familiar with Terraform's operating model and command line interface, and can usually adapt quickly to TFE's web interface.
 
-## The Optimal Terraform Workspace Structure
+## The Recommended Terraform Workspace Structure
```

Comment:
Good catch, I hate "optimal" too. I think that's a carry-over from previous versions of it.